### DISCUSSION TITLE
Error with workflow 3-1

### DISCUSSION THREAD

**Discussion by NZDivinity** [R3S3t9999/DoVi_Scripts#46] [General]
Hello.
I'm encountering the below error with workflow 3-1.
Unsure why, but suspect it could be due to my primary GPU being AMD (7900xtx)?
If so, I'd just be after confirmation.. and possibly a way to set my secondary (Nvidia) GPU to be used instead.
Orr I'm completely incorrect and would appreciate some guidance.

![image](https://github.com/user-attachments/assets/93c4caf7-ec1c-4081-aa46-6940a5d1afd7)

**Comment by R3S3t9999** [comment]
Madvr failed to create the shot list. It usually happens with TS or M2TS input and rarely with MKV files. 
if its a m2ts/ts, remux to mkv. If its a MKV, there's nothing you can do about it.

**Reply by NZDivinity** [reply]
Ah I see.. yeah the input is already MKV unfortunately.
How rare is this error with MKV?
I've tried with 3 different input files and they've all lead to the same error.
